packages/terra-collapsible-menu-view/tests/jest/CollapsibleMenuViewHyperlink.test.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+/* eslint-disable-next-line import/no-extraneous-dependencies */
+import { shallowWithIntl } from 'terra-enzyme-intl';
+import CollapsibleMenuViewHyperlink from '../../src/CollapsibleMenuViewHyperlink';
+
+describe('CollapsibleMenuViewHyperlink', () => {
+  // Snapshot Tests
+  it('should render a default component', () => {
+    const wrapper = shallowWithIntl(<CollapsibleMenuViewHyperlink text="Testing" href="https://www.oracle.com/" />);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  // Props
+  it('should merge custom props', () => {
```

Comment:
What does this test do? I'm confused at what the before/after would be.

Reply:
It was an existing test that the other collapsible menu view components used, but I think it doesn't really test anything so I'll remove it